nixos/modules/installer/netboot/netboot-base.nix

```diff
@@ -1,7 +1,7 @@
 # This module contains the basic configuration for building netboot
 # images
 
-{ lib, ... }:
+{ ... }:
```

Comment:
You can remove this line completely

Reply:
Alright. After further deliberation (and other opinions), i have decided to just make it a set. Should be marginally faster